tests/java/test-specs.yml

```diff
@@ -1,11 +1,19 @@
-description: End-to-end tests for java initcontainer
+description: End-to-end tests for Java initcontainer
 custom_test_key: tags.testKey
 scenarios:
   - description: This scenario will verify that a transaction is reported by the test app after a curl request
     before:
+      # Deploy test app chart
       - helm install test-java ./chart/ --set=scenarioTag="${SCENARIO_TAG}" -n default
+      - sleep 5
+      # Recreate test app pod to ensure instrumentation resource is used
+      - kubectl delete pods -l "app=test-app-java"
+      # Wait for pods to pass healthchecks
       - sleep 120
```

Comment:
It's interesting that Java has a 120 second sleep here whereas the other agents have a 5 second one...should they all be the same?

Reply:
Java added that awhile ago because their sample app needed more time to start up. I'm not going to be the one to change that